p2p/src/peers/mod.rs

```diff
@@ -374,6 +374,21 @@ impl Peers {
 
         true
     }
+
+    /// Remove address from the ice bucket
+    pub fn ice_peer_address_remove(&mut self, address: &SocketAddr) {
```

Comment:
For uniformity with the other buckets:
```suggestion
    pub fn remove_from_ice(&mut self, address: &SocketAddr) {
```